pkg/planner/core/find_best_task.go

```diff
@@ -53,6 +54,10 @@ var PlanCounterDisabled base.PlanCounterTp = -1
 
 var invalidTask = &RootTask{} // invalid if p is nil
 
+func init() {
```

Comment:
```planner/core``` has 4 ```init``` function. Should we merge those together?

Reply:
make sense